### PR TITLE
Log about webengine remote debugging

### PR DESCRIPTION
**Pull request by sethalves**
- log a line about the contents of QTWEBENGINE_REMOTE_DEBUGGING before exiting

### CODE REVIEW

**Comment by sethalves** [comment]
test this please

**Comment by jherico** [comment]
why are we logging it at shutdown instead of startup?

**Comment by ZappoMan** [comment]
@jherico because the crash happens on shutdown... so I want it to be closer to where the crash actually occurs...

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -1120,6 +1122,10 @@ void Application::aboutToQuit() {
 }
 
 void Application::cleanupBeforeQuit() {
+    // add a logline indicating if QTWEBENGINE_REMOTE_DEBUGGING is set or not
+    QString webengineRemoteDebugging = QProcessEnvironment::systemEnvironment().value("QTWEBENGINE_REMOTE_DEBUGGING", "false");
+    qCDebug(interfaceapp) << "QTWEBENGINE_REMOTE_DEBUGGING =" << webengineRemoteDebugging;
```
Comment (ZappoMan) [inline interface/src/Application.cpp]: Maybe we should actually make the message say... "-- Expected CRASH!" and/or the same message you added to the bugsplat tech support entry

**Comment by hifi-gustavo** [comment]
Build console output for commit: bda8761e00bfede783f80c5171e700db5ff5d68e available at: https://gist.github.com/b60f79b2095a0f6b3405cb1b3ee878df<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7579.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7579.dmg

File: interface/src/Application.cpp (hunk 1 above)
Comment (sethalves) [inline interface/src/Application.cpp]: I'm afraid that nobody will remember to go fix that logging once it's actually fixed.

**Comment by sethalves** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: bda8761e00bfede783f80c5171e700db5ff5d68e available at: https://gist.github.com/b60f79b2095a0f6b3405cb1b3ee878df<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7579.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7579.dmg

**Comment by sethalves** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: bda8761e00bfede783f80c5171e700db5ff5d68e available at: https://gist.github.com/b60f79b2095a0f6b3405cb1b3ee878df<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7579.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7579.dmg

**Comment by ZappoMan** [comment]
Testing Notes:
- Set the QTWEBENGINE_REMOTE_DEBUGGING  environment variable.
- Smoke test.
- Shut down  [expect a crash, since that's what happens]
-  check the hifi-log that gets uploaded to bug splat and see that the warning line is included in the log

**Comment by imgntn** [comment]
i've seen the crash window show up multiple times on both my home and work machine with this env var set ```Crash Report #2701
This is a known crash -- to work around it, unset QTWEBENGINE_REMOTE_DEBUGGING from your environment variables.```

**Comment by ZappoMan** [comment]
tested and confirmed that the log file includes the message

**Pull request merged**